src/main/java/com/mixpanel/android/viewcrawler/LayoutCaller.java

```diff
@@ -0,0 +1,40 @@
+package com.mixpanel.android.viewcrawler;
+
+import android.view.View;
+import android.widget.RelativeLayout;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class LayoutCaller {
+    public LayoutCaller(JSONArray args) throws JSONException {
```

Comment:
I'd prefer if this constructor didn't take JSON- it feels like this object could be independent of the protocol we use to create them.